docs/Self-Service-MOPs/Ad-Hoc-E2E-Job.md

```diff
@@ -0,0 +1,13 @@
+* Ad-Hoc e2e test run
+
+
+Job definition: https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/jenkins/osde2e/job-templates.yaml?ref_type=heads#L67
```

Comment:
Instead of a direct link to a specific line in the file, how about we just link to the file? This way as the file changes, we do not have to keep update the line number. Right now line 67 is pointing to one of the build parameters.